### ISSUE TITLE
Are 	WebFace4M and WebFace12M are the same as  WebFace260M or WebFace42M? Or are they different models?

### ISSUE THREAD

**Issue by martinenkoEduard** [mk-minchul/AdaFace#97]
Are 	WebFace4M and WebFace12M are the same as  WebFace260M or WebFace42M? Or are they different models?

**Comment by xxiMiaxx** [comment]
yes they are the same, according to their [paper](https://arxiv.org/abs/2103.04098) they collected 260M images (50 TB), however the entire dataset WebFace260M is not clean they are just raw from the internet, they are noisy, so they created clean subsets which are Webface42M the largest dataset (as far as I know) to train for FR, and WebFace12M and WebFace4M, these subsets are clean and ready to train on.
The large noisy one WebFace260M is an interesting research problem on its own.

models that are trained on the different subsets give different performances, the best performances are usually associated with more data ie webface42m.

**Comment by martinenkoEduard** [comment]
> 



> yes they are the same, according to their [paper](https://arxiv.org/abs/2103.04098) they collected 260M images (50 TB), however the entire dataset WebFace260M is not clean they are just raw from the internet, they are noisy, so they created clean subsets which are Webface42M the largest dataset (as far as I know) to train for FR, and WebFace12M and WebFace4M, these subsets are clean and ready to train on. The large noisy one WebFace260M is an interesting research problem on its own.
> 
> models that are trained on the different subsets give different performances, the best performances are usually associated with more data ie webface42m.

So it is worth trying to retrain adaFace on Webface42M, because it is bigger than WebFace12M?
